papergres_test.go

```diff
@@ -206,15 +166,8 @@ func TestCanCreateValidSchemaObjectFromDatbase(t *testing.T) {
 
 func TestCanGenerateValidInsertSql(t *testing.T) {
 	// setup()
-	sql := insertSQL(book, "paper")
-	fmt.Println(sql)
-}
-
-func TestCanGenerateValidInsertMultipleSql(t *testing.T) {
-	// setup()
-	sql, args := insertMultipleSQL(characters, "paper")
+	sql := insertSQL(book, "paper", false)
```

Comment:
What is the test case here? I don't understand what is being tested.

Reply:
Thanks for catching this. Missed adding the actual book object. Sent a fix